### PR TITLE
Add computer offline sprite override

### PR DESCRIPTION
**Pull request by NotMyFault**
This is likely a left over from https://github.com/jenkinsci/jenkins/pull/5663. Or something that has been forgotten.

### Proposed changelog entries

* Use offline node SVG icon instead of PNG

### Proposed upgrade guidelines

N/A

### Submitter checklist

- [X] Changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developer, depending on the change). [Examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)
  * Fill-in the `Proposed changelog entries` section only if there are breaking changes or other changes which may require extra steps from users during the upgrade

### Desired reviewers

@mention

### Maintainer checklist

Before the changes are marked as `ready-for-merge`: 

- [ ] There are at least 2 approvals for the pull request and no outstanding requests for change
- [ ] Conversations in the pull request are over OR it is explicit that a reviewer does not block the change
- [ ] Changelog entries in the PR title and/or `Proposed changelog entries` are correct
- [ ] Proper changelog labels are set so that the changelog can be generated automatically
- [ ] If the change needs additional upgrade steps from users, `upgrade-guide-needed` label is set and there is a `Proposed upgrade guidelines` section in the PR title. ([example](https://github.com/jenkinsci/jenkins/pull/4387))
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins-ci.org/issues/?filter=12146)).

### CODE REVIEW

File: core/src/main/java/org/jenkins/ui/icon/IconSet.java [hunk 1]
```diff
@@ -308,6 +308,7 @@ private static String toNormalizedIconUrl(String url) {
         icons.addIcon(new Icon("icon-grey icon-sm", "16x16/grey.png", Icon.ICON_SMALL_STYLE));
         icons.addIcon(new Icon("icon-text-error icon-sm", "16x16/text-error.png", Icon.ICON_SMALL_STYLE));
         icons.addIcon(new Icon("icon-text icon-sm", "16x16/text.png", Icon.ICON_SMALL_STYLE));
+        icons.addIcon(new Icon("icon-computer-user-offline icon-sm", "16x16/icon-computer-user-offline.png", Icon.ICON_SMALL_STYLE));
```
Comment (timja) [inline core/src/main/java/org/jenkins/ui/icon/IconSet.java:311]: seems you've just added png everywhere? changelog talks about svg

Comment (NotMyFault) [reply core/src/main/java/org/jenkins/ui/icon/IconSet.java:311]: Yeah, the icons weren't added to the icon set registry and therefore weren't overridden by the appropriate SVGs.

Comment (timja) [reply core/src/main/java/org/jenkins/ui/icon/IconSet.java:311]: if they are supposed to be overridden then just remove the .pngs.

Comment (NotMyFault) [reply core/src/main/java/org/jenkins/ui/icon/IconSet.java:311]: That'll work too. I wasn't too sure about removing them in favor of compat with older versions.

**Review by timja — APPROVED** [review]
tbh I can't see anywhere where the old icon is being displayed but this is good cleanup anywhere.

When I checked before and after there was no change for me

**Review by oleg-nenashev — APPROVED** [review]
Fine with me.

We may merge it in 24 hours if there is no negative feedback. Please see the [merge process documentation](https://github.com/jenkinsci/jenkins/blob/master/docs/MAINTAINERS.adoc#merge-process) for more information about the merge process

**Pull request merged**